src/test/java/org/opentripplanner/test/support/JsonAssertions.java

```diff
@@ -0,0 +1,21 @@
+package org.opentripplanner.test.support;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class JsonAssertions {
+
+  private static final ObjectMapper mapper = new ObjectMapper();
+
+  public static void assertEqualJson(String expected, String actual) {
```

Comment:
```suggestion
  /**
   * Take two JSON documents and reformat them before comparing {@code actual} with {@code expected}.
   */
  public static void assertEqualJson(String expected, String actual) {
```

Reply:
Automerge merged this PR so I added it to dev-2.x directly: https://github.com/opentripplanner/OpenTripPlanner/commit/7e68673a19657d3a9935403a8de4dd2a10b38346